include/oneapi/tbb.h

```diff
@@ -29,7 +29,7 @@
 #include "oneapi/tbb/blocked_range2d.h"
 #include "oneapi/tbb/blocked_range3d.h"
 #if TBB_PREVIEW_BLOCKED_RANGE_ND
```

Comment:
It seems like this macro should be removed.

Reply:
The patch does not remove the preview status; a separate patch is needed for this.

Reply:
OK, now the patch does remove the preview status.